### ISSUE TITLE
GoogleMusic plugin menu items not shown

### ISSUE THREAD

**Issue by nikclayton** [nikclayton/android-squeezer#108]
```
What steps will reproduce the problem?
1. Properly install the GoogleMusic plugin on your home Logitech Media Server (https://github.com/hechtus/squeezebox-googlemusic).
2. On Squeezer, select "My apps", then "Google Music".

What is the expected output? What do you see instead?
I expect to see the GoogleMusic plugin's menu items ("My Music", "All-Inclusive").
Instead I see no menu items at all. It is not possible to control the GoogleMusic plugin
with Squeezer.

When I use the Logitech Media Server's web interface, I can browse the GoogleMusic
plugin menu items and control the plugin. I have noticed that there are no icons on
the menu items that are missing, so maybe that is the problem?

What version of the product are you using? On what operating system?
Latest version of Squeezer, SqueezeBox server and GoogleMusic plugin.
```

Original issue reported on code.google.com by `shredzone` on 2014-11-16 10:52:34

**Comment by nikclayton** [comment]
https://github.com/nikclayton/android-squeezer/wiki/Apps-support-in-Squeezer

**Comment by kaaholst** [comment]
Fixed in 2.0.0
